srv/components/provisioner/scripts/cortx_prep.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+set -euE
+export LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/node_prep.log}"
+mkdir -p $(dirname "${LOG_FILE}")
+
+PRVSNR_ROOT="/opt/seagate/cortx/provisioner"
+localhost_ip="127.0.0.1"
+minion_id="srvnode-0"
+user_name=
+uid=
+group=
+gid=
+repo_url=
+ISO_CORTX_PATH="/opt/isos/cortx-*-single.iso"
+ISO_OS_PATH="/opt/isos/cortx-os-*.iso"
+nodejs_tar="http://cortx-storage.colo.seagate.com/releases/cortx/github/integration-custom-ci/centos-7.8.2003/custom-build-1969/3rd_party/commons/node/node-v12.13.0-linux-x64.tar.xz"
```

Comment:
Can this be handled in separate function to install non-cortx packages

Reply:
Done